section2/reflection.md

```diff
@@ -1,15 +1,26 @@
 
 3. In this section, we removed some of the supports that you had in Section 1. We didn't give the directions for how to run a file in node and made some directions a bit more vague, intentionally. How did that feel? What was uncomfortable about it? How did it support your learning?
+  I enjoyed the added challenge! It forced me to look some things up on my own, and I feel that I got a small sense of accomplishment when I found the answer! The unknown was a bit uncomfortable, but not too much so.
 
 4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.
+  * A conditional statement is a way for a computer to make a decision based on specific conditions - if/then/else.
+  * A daily conditional in my life could be the decision whether or not to eat a snack. I could evaluate my hunger, and if the answer is amIHungry === true, then I would 'log to my console' *eat a snack*! If the answer is false, I would 'log to my console' *ask again in one hour*!
+  * A conditional that I probably use daily in a web application could be on my podcast app. I often set a sleep timer as a way to indicate that it's time to change a task. For example, I'll wash dishes for ten minutes, and when the timer ends and the podcast stops playing, I can be done. I would guess that this conditional statement could look like 'is the timer done?' if the answer is yes, stop podcast. Or 'is the timer done?' if the answer is no, continue to play the podcast.
 
 5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
+  You add multiple `if` statements by following the first `if` statement with an `else if` statement and so on. The computer will go in order, and it will stop evaluating the conditions as soon as one evaluates to *true*. When this true statement is found, wherever it is in the conditional, the computer will then execute the associated block of code!
 
 6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
+  You could use the logical operators! The operator && will check to se if both conditions are met, and the operator || will check if either of the conditions are met.
 
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+7. What questions do you still have about `if` statements and/or functions?
+  I don't know if it's a question, but I would love to see how `if` statements work together! Can you run multiple statements inside of one another? Nests of statements?
```

Comment:
Great question! 

Yes, you absolutely can. I'd say they get difficult to read/write/update if bugs appear of functionality needs to change if they are nested more than 2 deep.